Sources/BuildSystemIntegration/ExternalBuildSystemAdapter.swift

```diff
@@ -178,6 +185,48 @@ actor ExternalBuildSystemAdapter {
     ).connection
   }
 
+  private func getConfigPath() -> AbsolutePath? {
+    var buildServerConfigLocations: [AbsolutePath?] = []
+
+    let workspacePath = projectRoot.appending(component: ".bsp")
+    buildServerConfigLocations.append(workspacePath)
```

Comment:
Can easily be one operation

```suggestion
    buildServerConfigLocations.append(projectRoot.appending(component: ".bsp"))
```

Same below.